packages/devtools_app/lib/src/screens/inspector/inspector_controller.dart

```diff
@@ -143,7 +140,7 @@ class InspectorController extends DisposableController
     serviceManager.consoleService.ensureServiceInitialized();
   }
 
-  void _handleConnectionStart(VmService service) {
+  void _handleConnectionStart(VmService? service) {
```

Comment:
Parameter is nullable because of handler signature

Reply:
nit: is a null service actually means the service is stopped, add a TODO to rename the API to be something more accurate.

Reply:
hm. Today I could change it. May be because of recent upgrade to next version of SDK.